be/extmap.c

```diff
@@ -865,34 +967,68 @@ be_emap_ksize(const void* k)
 static m0_bcount_t
 be_emap_vsize(const void* d)
 {
-	return sizeof(struct m0_be_emap_rec);
+	return sizeof(struct m0_be_emap_rec) +
+		((struct m0_be_emap_rec *)d)->er_cksum_nob;
 }
 
 static int
 emap_it_pack(struct m0_be_emap_cursor *it,
-	     void (*btree_func)(struct m0_be_btree *btree,
-				struct m0_be_tx    *tx,
-				struct m0_be_op    *op,
-			  const struct m0_buf      *key,
-			  const struct m0_buf      *val),
+             void (*btree_func)(struct m0_be_btree  *btree,
+			        struct m0_be_tx     *tx,
```

Comment:
[* style *] use tab.
[* style *] check alignment.
same for other parameters below

Reply:
Fixed